src/picknik_ur_base_config/objectives/looping_pick_and_place_object.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
```

Comment:
Does this need to be here?

If so should we have a config yaml for it so that we can at least run it in mock hw without a missing config file error?

Reply:
It does not -- site config is where it should be. Great catch.